e2e.Dockerfile

```diff
@@ -6,12 +6,13 @@ WORKDIR /src/app/
 COPY go.mod go.sum* ./
 RUN go mod download
 COPY . .
+ENV KO_DOCKER_REPO=ghcr.io/cosmos
 ENV PACKAGES curl make git libc-dev bash gcc linux-headers eudev-dev python3
 RUN apk add --no-cache $PACKAGES
 RUN CGO_ENABLED=0 make install
 
 # Add to a distroless container
-FROM distroless.dev/static:$IMG_TAG
+FROM distroless.dev/ko:$IMG_TAG
```

Comment:
This won't have the intended effect. This still uses `docker build` to build the image, this change only updates the base image the resulting image would be based on.

If you want to use `ko` to build instead of `docker build`, you'll need to replace invocations of `docker build` with `ko build ./cmd/gaiad`, which will replicate the steps of this Dockerfile (e.g., `go build`) and put the resulting binary on top of the same distroless static image you're already using.

As a benefit, `ko` doesn't require Docker to build, and also generates an SBOM by default. If you want some help with this, let me know and I'd be happy to help with a PR. https://ko.build/advanced/migrating-from-dockerfile/ might also be helpful (if not, let me know and I'll fix it!)

Reply:
@imjasonh , can you help make a pr to fix this ? Thank you!